src/Parser.js

```diff
@@ -94,6 +94,25 @@ export function parse (value, root, parent, rule, rules, rulesets, pseudo, point
 
 				index = offset = property = 0, variable = ampersand = 1, type = characters = '', length = pseudo
 				break
+			// \
+			case 92:
```

Comment:
I'm a little worried about the fact that there's nothing here to distinguish between an escape sequence in an "identifier" as opposed to somewhere else, but I can't think of a test case where that's going to be a problem.